dfp_external_storage/dfp_external_storage/doctype/dfp_external_storage/dfp_external_storage.py

```diff
@@ -258,6 +318,10 @@ def dfp_external_storage_upload_file(self):
 			else:
 				frappe.throw(error_msg)
 
+	def set_is_private(self):
```

Comment:
Hi @khoran sorry for the delay, I am checking the code to approve your PR, and I do not understand this method calling itself, could you explain to me please? :) Thanks!

Reply:
There was something in the parent set_is_private function that caused a problem, I don't recall right now what that problem was. So this function just checks if we are using S3, then don't run the parent function, otherwise run it.

Reply:
Just checked and not affecting security of files:

1. upload as private > changed to public > changed to private: everything works as expected
2. upload as public > changed to private > changed to public: everything works as expected

Removed that code:

1. upload as private > changed to public > changed to private: everything works as expected
2. upload as public > changed to private > changed to public: everything works as expected

@khoran I temporarely removed it from "develop" branch, and just commit some fixes. Could you test if it is working fine for you? Here is the commit:
https://github.com/developmentforpeople/dfp_external_storage/commit/bd5360ca199241f01137819d4731e57a571fded1

Thank you very much for your help @khoran :)